### PR TITLE
📖 DOC: Coverage plugin

### PR DESCRIPTION
**Pull request by MrCordeiro**
Explanation what coverage is and how the assignments use it.

### CODE REVIEW

File: 06_testing/coverage.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Test Coverage
+
+Test coverage is a software development metric that measures the how much of your code is covered by tests. It's typically expressed as a percentage, where higher percentages indicate more extensive testing.
```
Comment (rfmmonteiro99) [inline 06_testing/coverage.md]:
```suggestion
Test coverage is a software development metric that measures how much of your code is covered by tests. It's typically expressed as a percentage, where higher percentages indicate more extensive testing.
```

File: 06_testing/coverage.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+pip install pytest pytest-cov
+```
+
+Then run `pytest` with the `--cov` flag:
+
+```shell
+pytest tests/ --cov
+```
+
+> **Note**: Remember that a high coverage doesn't guarantee bug-free code, as it only shows which parts of the code are executed during testing, not if the tests are effective or meaningful. Don't writ unnecessary tests just to increase coverage.
```
Comment (rfmmonteiro99) [inline 06_testing/coverage.md]:
```suggestion
> **Note**: Remember that a high coverage doesn't guarantee bug-free code, as it only shows which parts of the code are executed during testing, not if the tests are effective or meaningful. Don't write unnecessary tests just to increase coverage.
```

File: 06_testing/coverage.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## HTML Reports
+
+Sometime you want to see exactly what lines are not being covered by tests. To answer that question, you can generate an HTML report.
```
Comment (rfmmonteiro99) [inline 06_testing/coverage.md]:
```suggestion
Sometimes you want to see exactly what lines are not being covered by tests. To answer that question, you can generate an HTML report.
```

**Review by rfmmonteiro99 — APPROVED** [review]
I think these are good additions to FAAST 😊 good work, @MrCordeiro!

**Comment by rfmmonteiro99** [comment]
@MrCordeiro I only left 3 comments about typos I saw on the text, in case you want to commit them to the files :)

**Review by jmpmorais — APPROVED** [review]
Everything seems good to me ! @MrCordeiro

**Pull request merged**